### ISSUE TITLE
lintignore should let you use an explanatory comment

### ISSUE THREAD

**Issue by armsnyder** [bflad/tfproviderlint#203]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

`commentignore.Analyzer` has very strict matching on `lintignore` comments, which unfortunately means the user can't add context to why something is being ignored.

Here are two use cases that I expected to work, but the ignore was..... ignored.

```go
//lintignore:R009 // panic is ok in init functions
panic(err)
```

```go
// FIXME
//lintignore:AT001
func TestAccGitlabDeployKey_import(t *testing.T) {
```

### References

* See golangci-lint's `nolint` directive, which is similar and lets you include an explanation:
   * https://github.com/golangci/golangci-lint/blob/2d4d503a7906915b84d09972ed79376bae4dcf81/docs/src/docs/usage/false-positives.mdx#nolint

**Comment by bflad** [comment]
Support for multiple ignore keys and the extra comment syntax (e.g. `//lintignore:R009 // panic is ok in init functions`) has been merged and will release with version 0.19.0 of tfproviderlint, shortly. 👍
